README.md

```diff
@@ -65,9 +65,9 @@ directly.
 
 ### To configure a unikernel before building:
 
-    $ mirage configure [config.ml] [--unix|--xen]
+    $ mirage configure [config.ml] [-t unix|-t xen]
```

Comment:
@yomimono Should this also be changed to `mirage configure [-f config.ml] [-t unix|-t xen]`?
i.e. adding the `-f` as per https://mirage.io/wiki/breaking-changes?